ci/scripts/rsync_code_down.sh

```diff
@@ -40,7 +40,7 @@ echo 'StrictHostKeyChecking no' >> /etc/ssh/ssh_config
 
 OUTPUT_DIR=${BASE_DIR}/geode-results
 
-ssh -i ${SSHKEY_FILE} geode@${INSTANCE_IP_ADDRESS} "cd geode && ./gradlew combineReports"
+ssh -i ${SSHKEY_FILE} geode@${INSTANCE_IP_ADDRESS} "bash -c 'cd geode; ./gradlew --no-daemon combineReports'"
 
-time rsync -e "ssh -i ${SSHKEY_FILE}" -ah geode@${INSTANCE_IP_ADDRESS}:. ${OUTPUT_DIR}/.
+time rsync -e "ssh -i ${SSHKEY_FILE}" -ah geode@${INSTANCE_IP_ADDRESS}:geode ${OUTPUT_DIR}/
```

Comment:
This changes the structure of what ends up in geode-results, yes? before geode-results would have a geode subdirectory containing everything and this would make everything be directly in geode-results. I don't know if this breaks anything upstream.

Reply:
It should stay the same as it's `geode` (the actual directory) and not `geode/` (the contents of the directory) being copied

Reply:
Kk